### PR TITLE
Remote app

### PR DESCRIPTION
**Pull request by robertla**
I have pulled down from the master branch of azure-powershell today. No conflicts and everything built.

This update removes the requirement for a collection to be domain joined if it wishes to use a VNet

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@robertla__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (robertla). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
@azuresdkci - Add to white list please

**Comment by amarzavery** [comment]
@azuresdkci - Test this please.

**Comment by ogail** [comment]
@azuresdkci add to whitelist

File: src/Common/Commands.ScenarioTest/RemoteAppTests/CreateCloudCollection.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Azure.Common.Authentication;
```
Comment (ogail) [inline src/Common/Commands.ScenarioTest/RemoteAppTests/CreateCloudCollection.cs]: Add copyright header

File: src/ServiceManagement/RemoteApp/Commands.RemoteApp/Vnet/VNetDeprecated.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.WindowsAzure.Management.RemoteApp;
```
Comment (ogail) [inline src/ServiceManagement/RemoteApp/Commands.RemoteApp/Vnet/VNetDeprecated.cs]: Add copyright headers

File: src/ServiceManagement/RemoteApp/Commands.RemoteApp/Vnet/VNetDeprecated.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+using System.Management.Automation;
+using System.Reflection;
+
+namespace Microsoft.WindowsAzure.Management.RemoteApp.Cmdlets
+{
+    public class VNetDeprecated : RdsCmdlet
+    {
+        protected override void ProcessRecord()
+        {
+            string message = String.Format("This cmdlet {0} has been deprecated. See x {1}",
```
Comment (ogail) [inline src/ServiceManagement/RemoteApp/Commands.RemoteApp/Vnet/VNetDeprecated.cs]: Move the message to project resources

**Comment by amarzavery** [comment]
@robertla - Can you please address the code review comments and fix the CI failures?

**Comment by robertla** [comment]
I have everything fixed but the CI failures. I will look at that over the weekend

**Comment by stankovski** [comment]
@robertla merge conflicts

**Comment by markcowl** [comment]
@robertla Closing for now, please reopen when you're ready to addr5ess feedback

**Pull request closed without merging**